### ISSUE TITLE
Get current placement at result submission

### ISSUE THREAD

**Issue by d0ty** [Engame-Coding-Society/engame-quiz#27]
The goal is to send back the placement of the player after he/she submitted its score.

We'll need a placement field in the response. 
We have a `get_top_players` function, maybe if you create a modified version of it and optimize for to find the index of the player, it would do the trick. The only important thing is, you need to do this, before the return statement and after the `db.collection("..").document("").set(data)` statement, because this inserts/updates the player's score.

Basically that's all. You can reach me in dc or by mention me in the comments below!
